python/setup.py

```diff
@@ -61,7 +61,16 @@ def get_tag(self):
 data_files = [
     ("", ["LICENSE.txt"]),
 ]
-platform_package_data = [os.environ["TRITON_PYBIND"]]
+
+# Type checking marker file indicating support for type checkers.
```

Comment:
We should add a `__version__` to the package:
![image](https://github.com/triton-inference-server/core/assets/21284872/87b333e3-be85-477f-b949-da1d79bfbdc9)

Reply:
Completely separately, if this is easy to do, we should also add it to the `tritonclient` package. There was an open github issue about it in the past.
![image](https://github.com/triton-inference-server/core/assets/21284872/c54191d9-79ad-4e8d-9c90-4e6aa8cc6544)

Reply:
@debermudez for context